packages/actionmenu/src/react/item.tsx

```diff
@@ -178,10 +179,10 @@ export const Item = forwardRef<HTMLLIElement, ItemProps>(
         tabIndex={!disabled ? -1 : undefined}
       >
         <Wrapper {...styles.item({ hasSubMenu })} aria-haspopup={!!nested}>
-          <div className={className} {...styles.inner()}>
+          <span className={className} {...styles.inner()}>
```

Comment:
Why the tag change?

Reply:
So that it's valid markup.

Divs are invalid here https://validator.w3.org/nu/#textarea

Apparently related section of the spec is 3.2.5.1.5 "phrasing content" https://www.w3.org/TR/2012/WD-html5-20120329/content-models.html#phrasing-content